### DISCUSSION TITLE
Migration of nested collections

### DISCUSSION THREAD

**Discussion by szulcus** [supabase/supabase#10003] [Questions]
Hi. So far I have used Firestore. I would like to try my adventure with supabase. Is it possible to migrate collections nested within documents? I did not notice such a possibility in the migration guide 🤔

**Comment by steve-chavez** [comment]
Hello,

> Is it possible to migrate collections nested within documents? 

Could you provide some sample data so we can look at the shape of the json?

**Reply by szulcus** [reply]
Sure, I have a collection "books", with the name, a photo of the book etc. Each of these documents has a "units" collection. There are also some basic data and another collection of "parts" with vocabulary.

- 📂 books
  - 📄 [book-id]
    - name: string
    - author: string
    - supportedLanguages: string[]
    - ...
      - 📂 units
        - 📄 [unit-id]
          - name: string
          - ...
            - 📂 parts
              - 📄 [part-id]
                - name: string
                - words: { obligatory: boolean; en: string[]; pl: string[]; ... }[]
                - ...
